src/js/components/Meter/utils.js

```diff
@@ -19,6 +19,25 @@ export const strokeProps = (color, theme) => {
   return result;
 };
 
+export const fillProps = (color, theme) => {
+  const result = {};
+  if (color) {
+    if (typeof color === 'object') {
+      result.fill = normalizeColor(color.color, theme);
+      // if (color.opacity) {
```

Comment:
Is this needed? or remove

Reply:
So the stroke seemed to support an opacity from the theme. Now that's it's a filled shape I'd think we'd need to still support that opacity now for the fill?

Reply:
Yes agree to continuing to support opacity